phpstan.neon

```diff
@@ -0,0 +1,3 @@
+parameters:
+  level: 0
+  bootstrap: %currentWorkingDirectory%/vendor/autoload.php
```

Comment:
Afaik the default composer  autoloader dont need to be configured

Reply:
OK, I copied this file from other repos.
I will try without this line.

Reply:
You are correct - that line is not needed. Removed it.